src/broker/Manager.h

```diff
@@ -182,18 +182,20 @@ class Manager : public iosource::IOSource
 	 * Peers advertise interest by registering a subscription to some prefix
 	 * of this topic name.
 	 * @param name the name of the event
+	 * @param ts the timestamp the event is intended to be executed
 	 * @param args the event's arguments
 	 * @return true if the message is sent successfully.
 	 */
-	bool PublishEvent(std::string topic, std::string name, broker::vector args);
+	bool PublishEvent(std::string topic, std::string name, double ts, broker::vector args);
```

Comment:
This is in the `zeek::` namespace, so we can't insert that ts parameter here. Same arguments as for `Call()`.

I'm also stumbling over putting `ts` right between `name` and `args`. `name` is invoked with `args`, so they have a close relationship, while `ts` seems to add some additional info about when. Might just be me reading too much into it :-)